views/dashboard.ejs

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="/css/dashboard.css">
+
+    <link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/toastify-js/src/toastify.min.css">
+    <link href='https://unpkg.com/boxicons@2.0.7/css/boxicons.min.css' rel='stylesheet'>
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <link rel="icon" type="image/png" href="/res/logo/logo_white.png">
+    <link href="https://code.jquery.com/ui/1.10.4/themes/ui-lightness/jquery-ui.css" rel="stylesheet">
+
+    <script src="https://code.jquery.com/jquery-1.10.2.js"></script>
+    <script src="https://code.jquery.com/ui/1.10.4/jquery-ui.js"></script>
+    <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/toastify-js"></script>
+
+</head>
+<body>
+<div class="sidebar">
+    <div class="logo-details">
+        <img class="bx bxl-c-plus-plus icon" src="/res/logo/logo_white.png"/>
+        <div class="logo_name">Genetix</div>
+        <i class='bx bx-menu' id="btn"></i>
+    </div>
+    <ul class="nav-list">
+        <li>
+            <a href="#" id="home">
+                <i class='bx bx-grid-alt'></i>
+                <span class="links_name">Dashboard</span>
+            </a>
+        </li>
+        <li>
+            <a href="#" id="calendar">
+                <i class='bx bxs-calendar'></i>
+                <span class="links_name">Calendar</span>
+            </a>
+        </li>
+        <li>
+            <a href="#" id="drawboard">
+                <i class='bx bxs-edit'></i>
+                <span class="links_name">Settings</span>
+            </a>
+        </li>
+        <li>
+            <a href="#" id="settings">
+                <i class='bx bx-cog'></i>
+                <span class="links_name">Settings</span>
+            </a>
+        </li>
+        <li class="profile">
+            <div class="profile-details">
+                <div class="name_job">
+                    <div class="name"></div>
+                    <div class="job"></div>
+                </div>
+            </div>
+            <form> <!-- TODO add action when clicking on button -->
+                <button type="submit"><i class='bx bx-log-out' id="log_out"></i></button>
```

Comment:
According to Silvan we should add a testuser as a placeholder. I assume that would be done somewhere here, right @BadrOutiti?